docs/4.3.yaml

```diff
@@ -86,4 +86,5 @@ nav:
     - Reference:
         - YAML: config_reference.md
         - CLI: cli-docs.md
-        - API: api_reference.md
\ No newline at end of file
+        - API: api_reference.md
+        - Metrics & Logs: metrics_logs_reference.md
```

Comment:
nit: we don't have info about logs on that page yet

Reply:
Removed from sidebar for now.